tests/check_licenses.py

```diff
@@ -37,6 +38,8 @@
     'ring',
     # All runtime code that we depend on is covered by an ISC-style license.
     'webpki',
+    # https://github.com/hsivonen/encoding_rs#licensing
```

Comment:
Please explain the actual situation in the comment, the link might die at some point in the future.

Reply:
Ok